src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java

```diff
@@ -245,4 +246,20 @@ else if (PatternUtils.ALPHANUMERIC.matcher(texture).matches()) {
         }
     }
 
+    /**
+     * Adds additional lores for the {@link ItemStack}.
+     *
+     * @param newLore
+     *                New lores to be added to the {@link ItemStack}
+     */
+    public void addLore(String... newLore) {
+        ItemMeta meta = getItemMeta();
```

Comment:
You should check .hasItemMeta() before this, otherwise you are computing wasteful objects.